save-format.md

```diff
@@ -561,11 +557,163 @@ eb: byte
 49a: int  - Hard Wins
 49e: int  - Special Wins
 
+4a2: int  - Sum of Score
 4aa: int  - Top Score
 # Seems to record Area as 8 once in CO. This
 # has impact on camp size.
 4ae: byte - Deepest Area
 4af: byte - Deepest Level
+
+288e: time - Sum of Time
+2896: time - Best Time
+```
+
+## Area - Death Count
+
+```
+8b2: int - 1-1
+8b6: int - 1-2
+8ba: int - 1-3
+8be: int - 1-4
+
+cae: int - 2-1
```

Comment:
Interesting. Does it not have separate counters for jungle vs volcana or temple vs tide pool?

Reply:
No, I think these are only here for the 'deadliest level' stat in the player profile. That page doesn't make a distinction so it simply doesn't store it.

Reply:
Good catch, its a signed 64 bit number, I'll update this field and also add the type to the "Types" section